pkg/config/setup/system_probe.go

```diff
@@ -439,6 +440,9 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnvAndSetDefault(join(gpuNS, "max_streams"), 100)
 	cfg.BindEnvAndSetDefault(join(gpuNS, "max_stream_inactivity_seconds"), 30) // 30 seconds by default, includes two checks at the default interval of 15 seconds
 
+	// Software Inventory
+	cfg.BindEnvAndSetDefault(join(swNS, "enabled"), false, "DD_SOFTWARE_INVENTORY_ENABLED")
```

Comment:
No need to define a env alias since it's the same as the setting name.
```suggestion
	cfg.BindEnvAndSetDefault(join(swNS, "enabled"), false)
```